README.md

```diff
@@ -1,6 +1,7 @@
 # Ansible Role: HAProxy
 
 [![Build Status](https://travis-ci.org/geerlingguy/ansible-role-haproxy.svg?branch=master)](https://travis-ci.org/geerlingguy/ansible-role-haproxy)
```

Comment:
Would you mind fixing the travis link whilst you're at it?

Reply:
Done

Reply:
4d33f32 - I used my Josh bot :)